boilerplate/scripts/webpack.config.babel.js

```diff
@@ -6,6 +6,9 @@ import pkg from "../package.json"
 import config from "./config.js"
 
 export default {
+  ...config.dev && {
+    devtool: 'cheap-module-eval-source-map',
```

Comment:
Any point why this value and not another?
webpack devtool values are really confusing.
Btw, it's weird that the linter doesn't warn for singles quotes...